src/custom-pv-components/pdb-navigation.js

```diff
@@ -17,25 +17,58 @@ const height = 40,
   };
 
 class ProtvistaPdbNavigation extends ProtvistaNavigation {
-  
+
   connectedCallback() {
     this.style.display = 'block';
     this.style.width = '100%';
     this.width = this.offsetWidth;
 
     this._offset = parseFloat(this.getAttribute('offset')) || 0;
+    this._length = parseInt(this.getAttribute('length'));
+    this._displaystart = parseInt(this.getAttribute('displaystart')) || (this._offset > 0) ? this._offset : 1;
```

Comment:
Is displayStart modeling a boolean? value 1 is then true? then this.getBooleanAttribute (which returns true/false)

Reply:
It kinda seems so at first glance but in the end, the 1 value is more like a min value.
`<protvista-pdb-track class="pvTrack" length="1273" layout="non-overlapping" height="44" displayend="982" style="display: block; width: 100%;" displaystart="30" highlightend="null" highlightstart="null">`

Reply:
But then a parenthesis is missing? `?` has more precedent than `||` so I think you want: `a || (...)`